addons/pos_restaurant/static/src/js/multiprint.js

```diff
@@ -169,6 +169,24 @@ screens.OrderWidget.include({
 
 var _super_order = models.Order.prototype;
 models.Order = models.Order.extend({
+    build_by_product_resume: function(){
```

Comment:
put this code in the function "build_line_resume", as it is the function, and build_line resume is not done